cfpb/consumerfinance.gov
# Migrate legacy `a-tag` to `a-tag-filter`

## Changes

- Migrate legacy `a-tag` to `a-tag-filter`
 

## How to test this PR

1. Check the following filters and see that the tags are not bolded like production, but are regular weight like what is in the DS

http://localhost:8000/rules-policy/regulations/search-regulations/results/?regs=1002&q=mortage
http://localhost:8000/data-research/prepaid-accounts/search-agreements/?prepaid_type=Government+benefits
http://localhost:8000/consumer-tools/educator-tools/youth-financial-education/teach/activities/?grade_level=6


## Screenshots

| Before | After  |
| ------ | ------ |
|  <img width="536" alt="Screenshot 2024-11-19 at 11 29 09 AM" src="https://github.com/user-attachments/assets/5c8674b7-267b-4da7-aba2-2fe7bc88ff30"> |    
<img width="540" alt="Screenshot 2024-11-19 at 11 28 58 AM" src="https://github.com/user-attachments/assets/583cb9b9-3f02-46f3-81d1-6f61e0da1b5f"> |

## Note

iRegs has special tags that truncate "Regulation" at mobile down to "Reg". Because the tags now use flexbox, the whole tag label needs a container. Otherwise flexbox adds a gap after Reg. Wrapping the label in a generic span makes the whole thing a child of the flexbox and fixes this.
